browser/src/neovim/NeovimInstance.ts

```diff
@@ -324,6 +325,16 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
         return this.command(`e! ${fileName}`)
     }
 
+    public openInitVim(): Promise<void> {
+        const customInitVimPath = configuration.getValue("oni.customInitVimPath")
+
+        if (customInitVimPath) {
+            return this.command(`e! ${customInitVimPath}`)
```

Comment:
I should probably refactor this to just use the open command, so the logic isn't duplicated.

Reply:
Good idea 👍